tests/tests.toml

```diff
@@ -313,6 +313,7 @@ asr = true
 [[test]]
 filename = "../integration_tests/test_builtin_str.py"
 asr = true
+c = true
```

Comment:
Does the example compile? If so, I would remove this line, and rather enable it in integration_tests. That's the ultimate test, and it doesn't slow down `./run_tests.py`. We should only use the `./run_tests.py` if we are testing some complicated feature where it matters how the C code is generated.